src/copy.h

```diff
@@ -288,7 +288,7 @@ static inline zend_function* pthreads_copy_internal_function(zend_function *func
 static zend_function* pthreads_copy_function(zend_function *function) {
 	zend_function *copy = zend_hash_index_find_ptr(&PTHREADS_ZG(resolve), (zend_ulong)function);
 	
-	if (copy) {
+	if (copy && copy->op_array.run_time_cache == NULL) {
```

Comment:
This change does not look related and it also looks testable. If this is the case, can you separate this out into a separate PR (with a test) to ensure that it is definitely applied?

Reply:
I think the bug that this change fixes was already resolved by the fix for #725. This was one of the patches I tested that resolved the bug.

Reply:
Oh right, thanks for the information. It looks like it is no longer possible for the run time cache to be `NULL` with the other fix then, so this change doesn't seem needed anymore.

Reply:
You guys are right, this part is superfluous. I will update the PR, thx.